addons/gamification/tests/test_challenge.py

```diff
@@ -184,6 +184,40 @@ def test_30_create_challenge_with_sum_goal(self):
             "Challenge failed to start",
         )
 
+    def test_send_report_in_ranking(self):
+        model = self.env['ir.model'].search([('model', '=', 'gamification.badge')])[0]
```

Comment:
We prefer `self.env['ir.model']._get_id('gamification.badge')` to retrieve model IDs.